ctlra/devices/ni_maschine_mikro_mk3.h

```diff
@@ -0,0 +1,147 @@
+/*
+ * Copyright (c) 2016, OpenAV Productions,
+ * Harry van Haaren <harryhaaren@gmail.com>
```

Comment:
We can/should add your copyright - @zewelor what can we add here to accredit you as the developer?